### PR TITLE
Change unstable_external_project to unstable-external_project

### PR DESCRIPTION
**Pull request by tristan957**
Fixes a Meson 0.64.0 warning.

Signed-off-by: Tristan Partin <tpartin@micron.com>

### CODE REVIEW

File: .github/workflows/builds.yaml [hunk 1]
```diff
@@ -231,12 +231,21 @@ jobs:
         run: |
           echo "CFLAGS=-Wno-deprecated ${CFLAGS}" >> "$GITHUB_ENV"
 
+        # We use the `unstable-external_project` Meson module to enable certain
```
Comment (tristan957) [inline .github/workflows/builds.yaml:234]: I vehemently disagree with the Meson direction here. Tried to plead my case. No dice. Oh well, now we get this hack!

Comment (tristan957) [reply .github/workflows/builds.yaml:234]: https://github.com/mesonbuild/meson/pull/11053

Comment (eli-schwartz) [reply .github/workflows/builds.yaml:234]: Merged with just the change to fatal-warnings, as originally discussed when you pled your case. :) This should do everything you need it to do.

Comment (tristan957) [reply .github/workflows/builds.yaml:234]: Right I will change this when 0.64.1 releases

**Review by alexttx — APPROVED** [review]

**Review by nabeelmmd — APPROVED** [review]

**Pull request merged**